core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dataprocModels.scala

```diff
@@ -112,3 +112,21 @@ object PropertyFilePrefix {
 
   def stringToObject: Map[String, PropertyFilePrefix] = values.map(v => v.toString -> v).toMap
 }
+
+// Dataproc statuses: https://googleapis.github.io/google-cloud-dotnet/docs/Google.Cloud.Dataproc.V1/api/Google.Cloud.Dataproc.V1.ClusterStatus.Types.State.html
+sealed trait DataprocClusterStatus extends EnumEntry
+object DataprocClusterStatus extends Enum[DataprocClusterStatus] {
+  val values = findValues
+
+  case object Creating extends DataprocClusterStatus // The cluster is being created and set up. It is not ready for use.
+
+  case object Deleting extends DataprocClusterStatus // The cluster is being deleted. It cannot be used.
+
+  case object Error extends DataprocClusterStatus // The cluster encountered an error. It is not ready for use.
+
+  case object Running extends DataprocClusterStatus // The cluster is currently running and healthy. It is ready for use.
```

Comment:
Is the comment above accurate given [Rob's Slack message from earlier today](https://broadinstitute.slack.com/archives/GB1VAM14J/p1584558817217800) (i.e. is a Dataproc cluster in `Running` status always ready for use if its underlying instances may be `Stopped`)?

Reply:
yea, they're accurate. this is direct mapping for gce instances, the explanation is from https://cloud.google.com/compute/docs/instances/checking-instance-status